### PR TITLE
Add values for all environment variables

### PR DESCRIPTION
**Pull request by rujche**
Add values for all environment variables. Use "xxx" as placeholder if the value is secret.

### CODE REVIEW

File: cli/azd/pkg/project/scaffold_gen.go [hunk 1]
```diff
@@ -499,18 +499,18 @@ func getServiceSpecByName(infraSpec *scaffold.InfraSpec, name string) *scaffold.
...
 func printHintsAboutUsePostgres(authType internal.AuthType,
 	console *input.Console, context *context.Context) error {
-	(*console).Message(*context, "POSTGRES_HOST")
-	(*console).Message(*context, "POSTGRES_DATABASE")
-	(*console).Message(*context, "POSTGRES_PORT")
-	(*console).Message(*context, "spring.datasource.url")
-	(*console).Message(*context, "spring.datasource.username")
+	(*console).Message(*context, "POSTGRES_HOST=xxx")
+	(*console).Message(*context, "POSTGRES_DATABASE=xxx")
+	(*console).Message(*context, "POSTGRES_POR=xxxT")
```
Comment (saragluna) [inline cli/azd/pkg/project/scaffold_gen.go:504]:
```suggestion
	(*console).Message(*context, "POSTGRES_PORT=xxx")
```

File: cli/azd/pkg/project/scaffold_gen.go [hunk 2]
```diff
@@ -553,38 +553,38 @@ func printHintsAboutUseMySql(authType internal.AuthType,
...
 
 func printHintsAboutUseRedis(console *input.Console, context *context.Context) {
-	(*console).Message(*context, "REDIS_HOST")
-	(*console).Message(*context, "REDIS_PORT")
-	(*console).Message(*context, "REDIS_URL")
-	(*console).Message(*context, "REDIS_ENDPOINT")
-	(*console).Message(*context, "REDIS_PASSWORD")
-	(*console).Message(*context, "spring.data.redis.url")
+	(*console).Message(*context, "REDIS_HOST=xxx")
+	(*console).Message(*context, "REDIS_POR=xxxT")
```
Comment (saragluna) [inline cli/azd/pkg/project/scaffold_gen.go:557]:
```suggestion
	(*console).Message(*context, "REDIS_PORT=xxx")
```

**Review by saragluna — CHANGES_REQUESTED** [review]

File: cli/azd/pkg/project/scaffold_gen.go (hunk 1 above)
Comment (rujche) [reply cli/azd/pkg/project/scaffold_gen.go]: Updated.

File: cli/azd/pkg/project/scaffold_gen.go (hunk 2 above)
Comment (rujche) [reply cli/azd/pkg/project/scaffold_gen.go]: Updated.

**Review by saragluna — APPROVED** [review]

**Pull request merged**